### PR TITLE
docs: fix release notes for v14.0.0 - remove notice "fixes 45350" from breaking changes

### PR DESCRIPTION
**Pull request by Platonn**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -134,7 +134,7 @@
   callback was executed for the first time. Now it's tracked until
   the task is explicitly cancelled, e.g  with `clearInterval(id)`.
   
-  fixes 45350
+  fixes [#45350](https://github.com/angular/angular/pull/45350)
```
Comment (gkalpak) [inline CHANGELOG.md:137]: Nit: The canonical URL is `.../issues/...` (although the current URL redirects to that).

```suggestion
  fixes [#45350](https://github.com/angular/angular/issues/45350)
```

Comment (gkalpak) [reply CHANGELOG.md:137]: Actually, this `Fixes ...` reference should not be part of the breaking change notice.  (It failed to be recognized and removed by our tooling due to missing the `#` before the issue number.)

Could you please remove this line altogether?

Comment (Platonn) [reply CHANGELOG.md]: Thank you @gkalpak for review!
I removed the notice `fixes...`. I also updated the commit message and PR title.

**Review by devversion — APPROVED** [review]

**Comment by alxhub** [comment]
This PR was merged into the repository by commit da0f53be7638e91e6ba260b6ea2a2835c41c1084.

**Pull request closed without merging**